data/db-schema/src/main/resources/net/corda/db/schema/crypto/migration/crypto-creation-v1.0.xml

```diff
@@ -156,5 +179,27 @@
             <column name="tenant_id"/>
             <column name="alias"/>
         </createIndex>
+        <createTable tableName="crypto_signing_key_material">
+            <column name="wrapping_key_alias" type="VARCHAR(64)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="wrapping_key_generation" type="VARCHAR(64)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="signing_key_tenant_id" type="VARCHAR(12)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="signing_key_id" type="CHAR(12)">
```

Comment:
Why `CHAR(12)`? I would just make this UUID.

Reply:
Let's go to UUID.